packages/connect/README.md

```diff
@@ -19,6 +19,10 @@ The main implementation of the light-client provider for a given substrate-based
 
 ## Using `@substrate/connect` for library authors
 
+/*
+Can this package be used in isolation to connect to a light client, and then it uses the discovery protocol behind the scenes to see whether it can use one there instead? Or should somebody always use PAPI (which IIRC implements its own decision making around when to use what, but cant recall)?
```

Comment:
the connect package will look for a light client provider via the discovery protocol. if it can't find one, then it will spawn a smoldot instance in the user's browser tab.

Reply:
Yup ok awesome, that's what it looked like it did :) So it can be used on its own, nice!